modules/end-user-guide/partials/proc_mounting-a-secret-as-a-file-into-a-workspace-container.adoc

```diff
@@ -7,23 +7,57 @@
 
 WARNING: On 
 ifeval::["{project-context}" == "che"]
-{platforms-name} older than v1.13 and
+{kubernetes} older than v1.13 and
 endif::[]
-OpenShift {ocp3-ver}, secrets mounted as file overrides volume mounts defined in the devfile.
+OpenShift {ocp3-ver}, secrets mounted as a file override volume mounts defined in the devfile.
 
-This section describes how to mount a secret from the user's {orch-namespace} as a file in single-workspace or multiple-workspace containers of {prod-short}.
+In {prod-short}, you can mount the {platforms-name} secret from your {orch-namespace} as a file in single-workspace or multiple-workspace containers.
 
 .Prerequisites
 
-* A running instance of {prod-short}. To install an instance of {prod-short}, see {link-installing-an-instance}.
+* A running instance of {prod-short}. See {link-installing-an-instance}.
 
 .Procedure
 
-. Create a new {platforms-name} secret in the {platforms-name} {orch-namespace} where a {prod-short} workspace will be created.
-
-* The labels of the secret that is about to be created must match the set of labels configured in `che.workspace.provision.secret.labels` property of {prod-short}. The default labels are:
-
-* `app.kubernetes.io/part-of: che.eclipse.org` 
-* `app.kubernetes.io/component: workspace-secret`:
+. Create a new secret in the {platforms-namespace} where a new workspace is deployed.
+. Match the secret's `labels` to the set of labels configured in the `che.workspace.provision.secret.labels` property of {prod-short}.
```

Comment:
Don't anthropomorphize by using the possessive. IBM SG, pg. 26.
```suggestion
. Match the `labels` of the secret to the set of labels configured in the `che.workspace.provision.secret.labels` property of {prod-short}.
```

Reply:
@rkratky Our IBM SG pagination is different. May I ask which IBM SG version are you using? I'm using ibm_style_062121_Q2_2021:

Use the 's construction for the possessive form when it provides the clearest and most direct wording.
✗ Less direct: If the object is a child node, set the isLeaf property of the object to true.
✓ More direct: If the object is a child node, set the object's isLeaf property to true.
Use an alternative construction for the possessive form when it provides the clearest and most direct wording.
Alternative constructions include making a possessive noun an adjective or using a possessive noun in a prepositional
phrase.
✗ The computer's keyboard's Alt key stopped working.
✓ The Alt key on the computer keyboard stopped working.

Reply:
@rkratky https://source.redhat.com/groups/public/ccs/ccs_folder/ibm_style_guide